src/reducers/mission.js

```diff
@@ -24,7 +24,7 @@ export default handleActions({
     const mission = {
       id: parseInt(mission_id),
       vehicleId: vehicle_id,
-      price: parseFloat(price),
+      price: price,
```

Comment:
I think there is no harm in leaving the old `parseFloat` here. At the very least it will be helpful when upgrading Missions and Mission Control and we're not sure what we'll get in `price`. Also improve backwards compatibility.

Reply:
Good point, added it back 👍